hibernate-reactive-core/src/test/java/org/hibernate/reactive/containers/PostgreSQLDatabase.java

```diff
@@ -11,7 +11,7 @@
 
 	public static PostgreSQLDatabase INSTANCE = new PostgreSQLDatabase();
 
-	public final static String IMAGE_NAME = "postgres:12-alpine";
+	public final static String IMAGE_NAME = "postgres:13.0";
```

Comment:
Do we need to be extra precise? Wouldn't `13` be good enough?

Reply:
Other than that, feel free to merge it if it passes the checks

Reply:
I prefer using specifi tags in CI, so that stuff won't start failing because of an upgrade and we have no clue about it.

Reply:
For example: our github actions was likely already testing on pgsql v 13 as it was configured to use the latest.

Would you have suspected that new failures could have been caused by t he fact that PostgreSQL 13 was just released last week?

But also, it means our readme was not correct, as it states we're testing with v 12.

Reply:
> Would you have suspected that new failures could have been caused by t he fact that PostgreSQL 13 was just released last week?

Maybe

I understand the argument but I think it's easier to test for the latest 13.x version than update the version each time. I don't expect to have to deal with an exception because of this very often .

Anyway, it's ok for me to merge. 

We have this other workflow that's testing with the latest ORM 5.4 version available: https://github.com/hibernate/hibernate-reactive/blob/master/.github/workflows/tracking-orm-5.4.build.yml

I guess I will add a comment to mention that it also using the latest db

Reply:
@blafond This is a fix for PostgresSQL, but could you create an issue and update the images for MySQL and DB2 as well, please?

Ping me if something is not clear.
Thanks